robottelo/cli/factory.py

```diff
@@ -943,7 +943,7 @@ def make_repository_with_credentials(options=None, credentials=None):
 
 @cacheable
 def make_role(options=None):
-    """Usage::
+    """Usage:
```

Comment:
```suggestion
    """
    Usage:
```
just to be consistent